src/search.c

```diff
@@ -554,13 +535,6 @@ static enum swupd_code do_search(struct manifest *MoM, char search_type, char *s
 
 			/* Determine the level of completion we've reached */
 			if (hit) {
-				if (scope == 'b') {
-					done_with_bundle = true;
```

Comment:
if you are removing the assignment of `done_wit_bundle` then it has no purpose in the code anymore. It can be removed from line 516 and 477